skeleton/backstage/template.yaml

```diff
@@ -229,8 +232,8 @@ spec:
           defaultBranch: ${{ parameters.branch }}
           isTekton: ${{ parameters.ciType === 'tekton' }}
           isJenkins: ${{ parameters.ciType === 'jenkins' }}
-    - id: fetch-ci-skeleton
-      name: Fetch CI Components
+    - id: fetch-source-ci-skeleton
+      name: Fetch Source CI Configuration
```

Comment:
Since we're renaming this, maybe we can drop `Configuration` altogether. It's kind of long which may look odd on the UI and the `fetch-source-skeleton` counterpart does not use the `Configuration` terminology. Let's make them consisten?